modules/core/src/main/java/org/apache/ignite/IgniteSystemProperties.java

```diff
@@ -930,9 +930,8 @@
     public static final String IGNITE_PART_DISTRIBUTION_WARN_THRESHOLD = "IGNITE_PART_DISTRIBUTION_WARN_THRESHOLD";
 
     /**
-     * When set to {@code true}, WAL will be automatically disabled during rebalancing if there is no partition in
```

Comment:
Condition about absent partitions in OWNING state is important for understanding. Please leave it javadoc.

Reply:
OK, did it